lib/plugins/create/templates/aws-nodejs-typescript/serverless.ts

```diff
@@ -24,7 +25,11 @@ const serverlessConfiguration: AWS = {
     },
     lambdaHashingVersion: '20201221',
   },
-  functions: { hello }
-}
+  // import the function via paths
+  functions: { hello },
+};
+
+// use an import only available via tsconfig-paths
+logger(JSON.stringify(serverlessConfiguration, null, 2));
```

Comment:
You're already demonstrating the use of paths for @functions. I don't see the benefit from this added log line.

One thing you could do to demonstrate it is moving the handler path computation in lambda.ts helper and import it using paths in the hello function configuration file. WDYT ?

Reply:
makes perfect sense, i've updated the PR